Maui: Updated UI in Geoprocessing and Graphics overlay samples

ZackAllen opened the pull request:
# Description

Updated UI for five samples in geoprocessing and graphics overlay categories.

## Type of change

- [ ] Bug fix

**Platforms tested on**:

<!--- Delete any that aren't needed -->

- [x] MAUI Windows
- [x] MAUI Android
- [ ] MAUI iOS

## Checklist

- [ ] Runs and compiles on all active platforms
- [ ] Legacy platforms still compile and run (if applicable)
- [ ] Branch is up to date with the latest main/v.next
- [ ] All merge conflicts have been resolved
- [ ] Self-review of changes
- [ ] There are no warnings related to changes
- [ ] No unrelated changes have been made to any other code or project files
- [ ] `sample_sync.py` runs without making changes
- [ ] Codemaid and XAML styler extensions have been run on every changed file
- [ ] Code is commented with correct formatting
- [ ] All variable and method names are good and make sense
- [ ] There is no leftover commented code
- [ ] Screenshots are correct size and display in description tab

nCastle1 reviewed (commented):
- Format coordinates would benefit from row spacing and the new form pattern
- List transformations has an issue on mac catalyst - the listview goes out of bounds and doesn't display properly
- Spatial operations - picker is outside container bounds 
<img width="252" alt="image" src="https://user-images.githubusercontent.com/29742178/192073039-40b7a709-9540-4274-9114-98cef6aea496.png">

- I'm not sure the changes to animate 3d graphic are an improvement overall (aside from the color changes) - were there problems with the previous layout? This sample has always been something of a special case

ZackAllen commented:
> * Format coordinates would benefit from row spacing and the new form pattern
> * List transformations has an issue on mac catalyst - the listview goes out of bounds and doesn't display properly
> * Spatial operations - picker is outside container bounds
> 
> <img alt="image" width="252" src="https://user-images.githubusercontent.com/29742178/192073039-40b7a709-9540-4274-9114-98cef6aea496.png">
> 
> * I'm not sure the changes to animate 3d graphic are an improvement overall (aside from the color changes) - were there problems with the previous layout? This sample has always been something of a special case

1. Updated Format coordinates  to use the new pattern.
2. Not fixed yet.
3. Updated sample width values and shortened label text.
4. I wanted to make this UI match the other samples with new Borders. I also had to move the UI panel after the inset mapview to allow UI to be usable in tight mobile horizontal situations.
